packages/server/src/controllers/UsersController.js

```diff
@@ -4,7 +4,135 @@ import UsersValidator from '../validators/UsersValidator'
 
 const { User, UserAddress } = Models
 
+// temp dummy models
+let Users = { }
+let UserEmails = { }
+let UserSessions = { }
+
+
 export default {
+  async details(req, res) {
+    let body = {
+      "emailExists": false,
+      "emailVerified": false,
+      "addressVerified": false,
+      "notificationsEnabled": false
+    }
+    Object.assign(body, Users[req.params.address])
```

Comment:
Any reason for not doing simply 
```js
Users[req.params.address] = body
```

Reply:
Actually it's the other way around :)
```js
body = Users[req.params.address]
```
Some keys ins `Users[req.params.address]` may not exist so it will be an empty return instead of `false`